src/pages/sponsors/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import './sponsors.css'
+import { graphql } from 'gatsby'
+import Layout from '../../components/Layout'
+import PageHeader from '../../components/PageHeader'
+import { isValidUrl } from '../../utils/index'
+
+const SponsorsPage = ({ data }) => {
+  const getImageUrl = companyName => {
+    return isValidUrl(companyName)
+      ? companyName
+      : 'https://res.cloudinary.com/chicagojs/image/upload/2018Sponsors/' + companyName
+  }
+
+  // const grabTieredSponsors = (tier) => {
+  // 	const sponsorsArray = data.allSponsorsDataJson.edges.find(data => data.node.title === tier);
+  // 	if(sponsorsArray.node.items.length > 0) {
+  // 		return sponsorsArray.node.items.map((i,index) => (
+  // 			<div className={`col-md-6 col-lg-3 sponsorCard} ${tier === 'gold' ? 'col-lg-4' : 'col-lg-2'}`}>
+  // 				<div className="card border-0 transform-on-hover cardImgContainer">
+  // 					<a className="lightbox" href={i.url} target="_blank">
+  // 						<img style={`{width: tier === 'gold' ? '100%' : '75%'},${mobileLogo}]`} src={getImageUrl(i.image)} alt={`${i.name}`} className="card-img-top" />
+  // 					</a>
+  // 				</div>
+  // 			</div>
+  // 		));
+  // 	}
+  // }
+
+  return (
+    <Layout title="Sponsors">
+      <PageHeader
+        title={'Sponsors'}
+        titleColor={'#ffffff'}
+        background={
```

Comment:
i think what we'll do instead is pull this info from the URL and have it map all the backgrounds and titles in 1 spot. What do you think?

PageHeader.js:

```
switch (route) {
  case "sponsors":
     return { backgroundUrl: '', title: "Sponsors" }
}
```